nixos/modules/hardware/video/nvidia.nix

```diff
@@ -285,7 +285,7 @@ in
             KERNEL=="nvidia_uvm", RUN+="${pkgs.runtimeShell} -c 'mknod -m 666 /dev/nvidia-uvm-tools c $$(grep nvidia-uvm /proc/devices | cut -d \  -f 1) 1'"
           '';
           hardware.opengl = {
-            extraPackages = [ nvidia_x11.out ];
+            extraPackages = [ nvidia_x11.out nvidia_x11.settings.libXNVCtrl ];
```

Comment:
I had another look at the derivation and it actually doesn't seem to depend on `nvidia_x11`, except through `nvidia_x11.settingsVersion`. This makes me question if we actually have to deploy this impurely rather than link directly